### ISSUE TITLE
Updating gitlab profile picture

### ISSUE THREAD

**Issue by oscrx** [mend/renovate-ce-ee#495]
Can support be added to update the renovate profile picture to a bot user?
They just implemented the functionality: https://gitlab.com/gitlab-org/gitlab/-/issues/356868

**Comment by rarkins** [comment]
Which Renovate profile? Isn't this feature for group access tokens which are created by users (you) and not us?

**Comment by oscrx** [comment]
Hmm, A group access token is indeed created by us/the user/the operator. But there is no option to customize that account in the Gitlab UI for now. Adding a gpg key signature to the bot user or changing the name are also on my wishlist for example. I don't know if that is possible already. Or even if it's in-scope for Renovate ce/ee

**Comment by oscrx** [comment]
Renovate as user account:
<img width="1115" alt="afbeelding" src="https://github.com/mend/renovate-ce-ee/assets/6731096/7f7534c6-6b4c-4cda-a40f-73989f8b5575">
Renovate as group token / bot account:
<img width="1191" alt="afbeelding" src="https://github.com/mend/renovate-ce-ee/assets/6731096/82b69654-6747-4f04-99ac-d48e0045d65b">

(We migrated to a different Gitlab instance where a dedicated user is no longer possible)

**Comment by rarkins** [comment]
I don't think this is possible for Renovate to influence or change - I would assume that an administrator needs to make this change
